test/util.js

```diff
@@ -85,11 +81,9 @@ function createNewSub({ id, provider }) {
     events.emit('notification', value)
   })
   // subscription uninstall method
-  function uninstall(cb) {
-    provider.sendAsync({ method: 'eth_unsubscribe', params: [id] }, (err, res) => {
-      if (err) return cb(err)
-      cb(null, res.result)
-    })
+  async function uninstall(cb) {
+    const res = await provider.request({ method: 'eth_unsubscribe', params: [id] });
+    cb(null, res)
   }
```

Comment:
we can replace with
```suggestion
async function uninstall() {
    return await provider.request({ method: "eth_unsubscribe", params: [id] });
  }
```

and just return
```
  return {
    id,
    events,
    uninstall,
  };
```